frontend/src/utils/account-send.ts

```diff
@@ -0,0 +1,179 @@
+import { isAddress, keccak256, toUtf8Bytes, BigNumber, hexZeroPad, getAddress } from 'src/utils/ethers';
+import localforage from 'localforage';
+import { toAddress, lookupAddress } from 'src/utils/address';
+import { LOCALFORAGE_ACCOUNT_SEND_KEY, MAINNET_PROVIDER } from 'src/utils/constants';
+import { Web3Provider } from 'src/utils/ethers';
+
+// Send data that is encrypted and stored in local storage
+type AccountDataToEncrypt = {
+  address: string;
+  advancedMode: boolean;
+  usePublicKeyChecked: boolean;
+  pubKey: string; // can be full or partial pubKey
+};
+
+// Used to create key which is used to encrypt account data
+type KeyData = {
+  encryptionCount: number;
+  viewingKey: string;
+};
+
+// Encrypted values that are stored in local storage
+type EncryptedAccountSendData = {
+  encryptedAddress: string;
+};
```

Comment:
Yes, but this type is meant to be the actual encrypted field in the localstorage object. We take everything in `AccountDataToEncrypt` and encrypt it in that single field.

It is probably clearer if the field is renamed. How about `sendCiptertext`? And I can modify the comment to say value instead of values.

Reply:
Renamed to accountSendCiphertext and can change if you have a better suggestion https://github.com/ScopeLift/umbra-protocol/pull/494/commits/cf26d88cbd0f5345920dd4fb66bd49550dbb1aff

Reply:
I like that, thanks!